modules/update-upgrading-cli.adoc

```diff
@@ -129,6 +129,12 @@ date, where the newest update is first in the list. Updates in the history have
 state `Completed` if the rollout completed and `Partial` if the update failed
 or did not complete.
 +
+You can use the `watch` command to follow the status:
```

Comment:
We aren't supposed to add new jq commands.

Reply:
Oh! I forgot. This came up recently in our team meeting. I am going to close the PR and bug then. 
@vikram-redhat FYI

Reply:
The reporter of this BZ is not sure if there's an alternative command we can use.
@sferich888 Are you able to suggest one?

Reply:
`oc get clusterversion -o jsonpath='{range .items[0].status.history[*]}Completion Time: {.completionTime} Version: {.version} State: {.state}{"\n"}{end}' -w`

This gets ride of the `watch` command and the `jq` command. 

- Note: `-w` here will only show you state changes, from OpenShift. So we may not want to include this in the docs. 
   - That said, droping the watch command entirely may not be a bad idea.

Reply:
@gpei are you able to verify this updated command by Eric?

@ahardin-rh sorry, I missed that it was asking for a jq based command.

Reply:
@vikram-redhat It works for me

Reply:
Thank you, all! We have  ` oc get clusterversion -o json|jq ".items[0].status.history" ` already in this section of the docs, so I am going to leave that as-is and close this PR to drop mention of the watch command, then close this BZ.